src/server.rs

```diff
@@ -3818,13 +3765,12 @@ impl Completable for PackageResult {
         let mut insert_text = self.name.clone();
 
         if imports
-            .clone()
-            .into_iter()
-            .map(|x| x.path)
-            .any(|x| x == self.full_name)
+            .iter()
+            .map(|x| &x.path)
+            .any(|x| *x == self.full_name)
```

Comment:
Okay, but `clone().into_iter()` should be a clippy lint that says "hey, maybe you don't need to consume the iterator..." 😂